baseplate/lib/thrift_pool.py

```diff
@@ -100,6 +101,7 @@ def thrift_pool_from_config(
             "timeout": config.Optional(config.Timespan, default=config.Timespan("1 second")),
             "max_connection_attempts": config.Optional(config.Integer),
             "max_retries": config.Optional(config.Integer),
+            "queue": config.Optional(config.String, default="lifo"),
```

Comment:
Is this actually something we want to vary via configuration? i.e. it's changing from one environment to the next? I think we'd probably just be OK with the queue being a hard-coded aspect of the application.

Reply:
My thought was that since this is per-pool, it might make sense to allow changing only one downstream pool

Reply:
That's important, for sure. It will still be configurable per-pool if we leave it out of config parsing. In the wire-up function it'd look like:

```python
baseplate.configure_context({
    "foo_service": ThriftClient(FooService.Client),  # uses the default LifoQueue
    "bar_service": ThriftClient(BarService.Client, queue=FifoQueue),  # overridden for just this pool
})
```

Reply:
Sure, that seems ok to me.